ui/lesson/Terminal.tsx

```diff
@@ -17,6 +17,14 @@ export default function Terminal({ success, lines, next, onChange }) {
   const handleInputChange = (event) => {
     const newInput = event.target.value
     setInput(newInput)
+    const answerText = event.target.value
+    if (answerText.match(/.*\n$/)) {
+      onChange(input)
+      setCommandHistory([...commandHistory, input])
+      setHistoryIndex(historyIndex + 1)
+      setCurrentIndex(historyIndex + 1)
+      setInput('')
```

Comment:
This is due to the unusual behavior of the default android keyboard not collecting keycodes on keydown